dhtnew.h

```diff
@@ -92,7 +92,7 @@ class DHTNEW
 private:
   uint8_t  _dataPin = 0;
   uint8_t  _wakeupDelay = 0;
-  uint8_t  _type = 0;
+  int      _type = 0;
```

Comment:
No need (yet) to change this to an **int**
assume it is a remaining artefact of the discussion.